server/integration_tests/test_data/inappropriate_query/query_1/chart_config.json

```diff
@@ -10,5 +10,7 @@
     "name": "",
     "place_type": ""
   },
-  "userMessage": "Sorry, could not complete your request."
+  "userMessage": {
```

Comment:
What does the client read?  Wondering how this was working before

Reply:
the client reads userMessage.msg. The client actually wasn't reading the userMessage from "abort" cases & just using a DEFAULT_MESSAGE set on the client side

Reply:
Good find